src/core/connection-parameters.cpp

```diff
@@ -59,4 +61,38 @@ connection_parameters::connection_parameters(std::string const & fullConnectStri
 
     factory_ = &dynamic_backends::get(backendName);
     connectString_ = connectString;
+    parse();
+}
+
+void connection_parameters::parse()
+{
+    std::stringstream ssconn(connectString_);
+    while (!ssconn.eof() && ssconn.str().find('=') != std::string::npos)
+    {
+        std::string key, val;
+        std::getline(ssconn, key, '=');
+        std::getline(ssconn, val, ' ');
+
+        char delim = 0;
```

Comment:
@vadz copy-paste code from one of the parser/splitter (sqlite3 session to be precise). Not my invention.
So if you think it should be changed - feel free to do so.